kit/src/lib/FrameworkContentBlock.svelte

```diff
@@ -0,0 +1,77 @@
+<script lang="ts">
+    import type { SvelteComponent } from "svelte";
+    import type { Framework } from "./types";
+
+    import { onMount} from "svelte";
+    import { getFrameworkStore } from "./stores";
+	import IconPytorch from "./IconPytorch.svelte";
```

Comment:
indent (also try using tabs instead of spaces, you can configure tab display width on github & in vs code)